hexhamming/python_hexhamming.cc

```diff
@@ -172,7 +172,7 @@ static PyObject * hamming_distance_byte_wrapper(PyObject *self, PyObject *args)
  * @param args      Python arguments for `check_hexstrings_within_dist` interface
  *                  - `string1` -- hex string
  *                  - `string2` -- hex string
- * @returns
+ * @returns			True if the hamming distance is less or equal to max_dist, False otherwise.
```

Comment:
I think the spacing rendered as tabs, so just changing it to spaces here to stay in-line with the other docs.
```suggestion
 * @returns         True if the hamming distance is less or equal to max_dist, False otherwise.
```